src/ngMessageFormat/messageFormatParser.js

```diff
@@ -189,7 +189,7 @@ MessageFormatParser.prototype.ruleInsideString = function ruleInsideString() {
         position.line, position.column, this.text);
   }
   var chars = match[0];
-  if (match == this.stringQuote) {
+  if (match[0] === this.stringQuote) {
```

Comment:
Nitpick: Either use `chars` or remove it (in the line above), since it's not used elsewhere.